hawkular-command-gateway-war/src/main/java/org/hawkular/cmdgw/ws/command/feed/ExportJDRResponseCommand.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.cmdgw.ws.command.feed;
+
+import java.util.Collections;
+import java.util.Map;
+
+import org.hawkular.bus.common.BasicMessage;
+import org.hawkular.bus.common.BasicMessageWithExtraData;
+import org.hawkular.bus.common.BinaryData;
+import org.hawkular.bus.common.ConnectionContextFactory;
+import org.hawkular.bus.common.Endpoint;
+import org.hawkular.bus.common.MessageProcessor;
+import org.hawkular.bus.common.producer.ProducerConnectionContext;
+import org.hawkular.cmdgw.api.ExportJDRResponse;
+import org.hawkular.cmdgw.ws.Constants;
+import org.hawkular.cmdgw.ws.MsgLogger;
+import org.hawkular.cmdgw.ws.command.Command;
+import org.hawkular.cmdgw.ws.command.CommandContext;
+
+/**
+ * @author Juraci Paixão Kröhling
+ */
+public class ExportJDRResponseCommand implements Command<ExportJDRResponse, BasicMessage> {
+    public static final Class<ExportJDRResponse> REQUEST_CLASS = ExportJDRResponse.class;
+
+    @Override
+    public BasicMessageWithExtraData<BasicMessage> execute(ExportJDRResponse response, BinaryData binaryData,
+                                                           CommandContext context)
+            throws Exception {
+
+        String status = response.getStatus();
+        String msg = response.getMessage();
+        MsgLogger.LOG.infof("Export JDR completed. Status=[%s], Message=[%s]", status, msg);
+
+        // determine what UI client needs to be sent the message
+        String uiClientId;
+
+        // TODO: how do we get this?
```

Comment:
No, the appropriate solution is yet to be defined. A discussion about it is happening on the -dev and on HAWKULAR-498 . In fact, this code is mostly what there's already present on the existing ExecuteOperation.